src/frontend/csrs-portal/src/app/auth/auth-config.module.ts

```diff
@@ -1,34 +1,48 @@
+/* eslint-disable arrow-body-style */
+import { HttpClient } from '@angular/common/http';
 import { NgModule } from '@angular/core';
-import { AuthModule, LogLevel } from 'angular-auth-oidc-client';
+import { AuthModule, LogLevel, StsConfigHttpLoader, StsConfigLoader } from 'angular-auth-oidc-client';
+import { map } from 'rxjs/operators';
 
+export const httpLoaderFactory = (httpClient: HttpClient) => {
+  const config$ = httpClient.get<any>(`assets/oidc.json`).pipe(
+    map((customConfig: any) => {
+      return {
+        triggerAuthorizationResultEvent: true,
+        postLoginRoute: customConfig.postLoginRoute,
+        logLevel: LogLevel.Debug,
```

Comment:
Hey @skiselev1 
What do you feel about changing it per environment?
Do you think the mode should be DEBUG in prod?
Thanks Sergey!

Reply:
Thank you! Completely agree. LogLevel setting value is updated by depending on environment.

Reply:
it was replaced by line:  logLevel: environment.production ? LogLevel.None : LogLevel.Debug,